src/main/java/com/google/sps/servlets/MenteeSignupServlet.java

```diff
@@ -49,7 +49,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String password = "";
 
     // Set up query to insert new user into database.
-    String query = "INSERT INTO User (fname, lname, username, email, major_id, is_mentor) "
+    String query = "INSERT INTO User (first_name, last_name, username, email, major_id, is_mentor) "
         + "VALUES (?, ?, ?, ?, ?, ?)";
 
     try {
```

Comment:
This is getting to be a very long method. Consider ways  you can break this up into helper methods (my first intuition is the stuff inside the the try and catch blocks)

Reply:
This method was the same as part of the method in MentorSignupServlet.java, so I created a Utility method they both use.